torch/fx/experimental/validator.py

```diff
@@ -370,10 +379,13 @@ def div(self, numerator: z3.ArithRef, denominator: z3.ArithRef) -> z3.ArithRef:
         def pow(self, base: z3.ArithRef, exp: z3.ArithRef) -> z3.ArithRef:
             return self._ops.pow(base, exp)
 
+        def pow_by_natural(self, base: z3.ArithRef, exp: z3.ArithRef) -> z3.ArithRef:
+            return self._ops.pow(base, exp)
+
         def mod(self, p: z3.ArithRef, q: z3.ArithRef) -> z3.ArithRef:
             return self._ops.mod(p, q)
 
-        def round(self, number: z3.ArithRef, ndigits: Optional[z3.ArithRef] = None) -> z3.ArithRef:
```

Comment:
@albanD There's something wrong with BC lint here. This is not public API surface. Am I supposed to rename this module to `_validator` or something?

Reply:
bc linter would be @kit1980 validator, not the public API-realted tests.

> This is not public API surface.

From the file, it does look like public API per the definition of public API from https://github.com/pytorch/pytorch/wiki/Public-API-definition-and-documentation